### PR TITLE
Python 3.12 switch

### PR DESCRIPTION
**Pull request by mgorny**
CC @gentoo/python

### CODE REVIEW

File: app-portage/gpyutils/files/implementations.txt [hunk 1]
```diff
@@ -20,8 +20,8 @@ python3_7		dead	3.7
 python3_8		dead	3.8
 python3_9		dead	3.9
 python3_10		supported	3.10
-python3_11		current	3.11
-python3_12		supported	3.12
+python3_11		supported	3.11
```
Comment (mgorny) [inline app-portage/gpyutils/files/implementations.txt:23]: I'm wondering if we should mark 3.10+3.11 as "old". It feels kinda weird to have CPy 3.10 as "old" but PyPy 3.10 as "supported", though.

Comment (mgorny) [reply app-portage/gpyutils/files/implementations.txt:23]: …or maybe 3.10+3.11 as "old", 3.12 "current", 3.13 + pypy3 "experimental".

Comment (arthurzam) [reply app-portage/gpyutils/files/implementations.txt:23]: > …or maybe 3.10+3.11 as "old", 3.12 "current", 3.13 + pypy3 "experimental".

I prefer that actually.

Comment (mgorny) [reply app-portage/gpyutils/files/implementations.txt]: Yeah, switched to that instead.

**Review by arthurzam — APPROVED** [review]

**Pull request closed without merging**